package.json

```diff
@@ -10,7 +10,7 @@
 		"test": "jest"
 	},
 	"license": "MIT",
-	"optionalPeerDependencies": {
+	"peerDependencies": {
```

Comment:
The meta tag is missing though. `peerDependencies` will by default install everything therein for the client project, even if they have no desire to use one of them.

Please add `peerDependenciesMeta` as per discussion in the pertaining issue.

Reply:
@mreiche : FYI: @conradstrassburger went to vacation and handed this over to me yesterday.

I took the liberty to also move `peerDependencies` below `devDependencies` so the big-yet-mostly-irrelevant `peerDependenciesMeta` block can stay with the `peerDependencies` block it modifies without splitting the continous list of <whatever-kind-of>-dependencies.